doc/user/content/connect-sources/postgres-amazon-rds.md

```diff
@@ -0,0 +1,395 @@
+
+1. Use the [`CREATE CLUSTER`](/sql/create-cluster/) command to create the new cluster and replica:
+
+    ```sql
+    CREATE CLUSTER ingest_rds_postgres REPLICAS (r1 (SIZE = 'medium'));
+    ```
+
+    At first, it's recommended to use a `medium` [size](/sql/create-cluster-replica/#sizes) replica or larger. This will help Materialize more quickly take an initial snapshot of the tables in your publication. Once the snapshot is finished, you'll right-size the cluster replica.
+
+1. Use the [`SHOW CLUSTER REPLICAS`](/sql/show-cluster-replicas/) command to check the status of the replica:
+
+    ```sql
+    SHOW CLUSTER REPLICAS WHERE cluster = 'ingest_rds_postgres';
+    ```
+    <p></p>
+
+    ```noftm
+           cluster       | replica |  size  | ready
+    ---------------------+---------+--------+-------
+     ingest_rds_postgres | r1      | medium | t
+    (1 row)
+    ```
+
+    Once the replica is ready (`ready=t`), move on to the next step.
+
+## Step 5. Start ingesting data
+
+With your database network configured and your source cluster created in Materialize, you can now connect Materialize to your PostgreSQL database and start ingesting data. The exact steps depend on your networking configuration, so start by selecting the relevant option.
+
+{{< tabs >}}
+
+{{< tab "Allow Materialize IPs">}}
+
+
+1. In [step 2](#step-2-create-a-publication), you created a `materialize` user in your PostgreSQL database. Use the [`CREATE SECRET`](/sql/create-secret/) command to store this user's password as a secrete in Materialize:
+
+    ```sql
+    CREATE SECRET pgpass AS '<PASSWORD>';
+    ```
+
+1. Use the [`CREATE CONNECTION`](/sql/create-connection/) command to create a connection object with access and authentication details for Materialize to use:
+
+    ```sql
+    CREATE CONNECTION pg_connection TO POSTGRES (
+      HOST '<host>',
+      PORT 5432,
+      USER 'materialize',
+      PASSWORD SECRET pgpass,
+      SSL MODE 'require',
+      DATABASE '<database>'
+      );
+    ```
+
+    Replace `<host>` with the your RDS endpoint. To find your RDS endpoint, select your database in the RDS Console, and look under **Connectivity & security**.
+
+    Replace `<database>` with the name of the database containing the tables you want to replicate to Materialize.
+
+1. Use the [`CREATE SOURCE`](/sql/create-source/) command to connect Materialize to your PostgreSQL database and start ingesting data from the publication you created in PostgreSQL earlier:
+
+    ```sql
+    CREATE SOURCE mz_source
+      IN CLUSTER ingest_rds_postgres
+      FROM POSTGRES CONNECTION pg_connection (PUBLICATION 'mz_source')
+      FOR ALL TABLES;
+    ```
+
+    To ingest data from specific schemas or tables in your publication, change `FOR ALL TABLES` to `FOR SCHEMAS (<schema1>,<schema2>)` or `FOR TABLES (<table1>, <table2>)`, respectively.
```

Comment:
```suggestion
    To ingest data from specific schemas or tables in your publication, use `FOR SCHEMAS (<schema1>,<schema2>)` or `FOR TABLES (<table1>, <table2>)` instead of `FOR ALL TABLES`, respectively.
```